ansible/roles/mn-evo-services/tasks/tendermint.yml

```diff
@@ -9,29 +9,37 @@
     owner: '{{ tendermint_uid }}'
     group: '{{ tendermint_gid }}'
     mode: 0750
-  register: tendermint_home_dir
 
-- name: initialize Tendermint node
-  command: docker-compose run --no-deps --rm --entrypoint='/usr/bin/tenderdash' tendermint init
-  args:
-    chdir: '{{ mn_evo_services_path }}'
-  when: tendermint_home_dir.changed
-
-- name: get Tendermint node ID
-  command: docker-compose run --no-deps --rm --entrypoint='/usr/bin/tenderdash' tendermint show-node-id
-  args:
-    chdir: '{{ mn_evo_services_path }}'
-  register: tendermint_node_id
+- name: create tendermint config dir
+  file:
+    path: '{{ mn_evo_services_path }}/tendermint/config'
+    state: directory
+    owner: '{{ tendermint_uid }}'
+    group: '{{ tendermint_gid }}'
+    mode: 0750
 
-- name: get Tendermint validator key
-  command: cat {{ mn_evo_services_path }}/tendermint/config/priv_validator_key.json
-  register: tendermint_validator_key
+- name: create tendermint data dir
+  file:
+    path: '{{ mn_evo_services_path }}/tendermint/data'
+    state: directory
+    owner: '{{ tendermint_uid }}'
+    group: '{{ tendermint_gid }}'
+    mode: 0750
 
-- set_fact:
-    tendermint_node_id: '{{ tendermint_node_id.stdout }}'
+- name: create Tendermint node_key.json
```

Comment:
you can use `loop` to template many files https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html